lib/log-events-processor.js

```diff
@@ -92,3 +104,24 @@ export const processor = (job, done) => {
     .then(saveRefetchedToken(job, done))
     .catch(handleCloudWatchLogsError(done))
 }
+
+const client = new Redis(process.env.REDIS_URL);
+const subscriber = new Redis(process.env.REDIS_URL);
+
+// cf. https://github.com/OptimalBits/bull/blob/master/PATTERNS.md#reusing-redis-connections
+export const getQueueCreateClient = (client, subscriber) => {
+  return (type) => {
+    switch (type) {
+      case 'client':
+        return client
+      case 'subscriber':
+        return subscriber
+      default:
+        return new Redis()
```

Comment:
Dans ce cas là, est-ce qu'il ne faut pas :

```suggestion
        return new Redis(process.env.REDIS_URL)
```

Reply:
Voire, même :

```suggestion
        return getRedis()
```

Reply:
Ou, sinon, si cette partie ne sert pas actuellement, lever une exception à la place ?

Reply:
J'avoue que je ne suis pas encore familière avec Bull. Je ne sais pas s'il faut utiliser l'instance mémoizée de Redis pour le cas par défaut. S'il dans son fonctionnement interne, il ferme cette connexion, mes appels suivants pour le `nextSequenceToken` ne répondront plus.

> Ou, sinon, si cette partie ne sert pas actuellement, lever une exception à la place ?

Je ne sais pas exactement comment le `switch` est utilisé par Bull. La [documentation](https://github.com/OptimalBits/bull/blob/master/PATTERNS.md#reusing-redis-connections) donne juste cette implémentation. Penses-tu qu'il faille que je creuse ? Ou je laisse tel quel pour la PR ?

Reply:
Ouais, bien vu.

Dans le doute, laisse tel-quel mais en passant quand même `REDIS_URL`, j'ai l'impression que c'est un oubli sur leur doc.

Reply:
Yes. Je creuserai plus tard.

La modif est faite pour le `REDIS_URL`.